getting-started/templates/pilot-sizing.yaml

```diff
@@ -145,6 +152,16 @@ dataframeservice:
       cpu: 4
       memory: 31000
       volumeSize: "50Gi"
+      engines:
```

Comment:
Do we need to explicitly set the engines here? `["default", "iceberg"]` is the default from the values file

Reply:
Nope! I've removed this.